samples/snippets/bqml_getting_started_test.py

```diff
@@ -164,5 +156,40 @@ def test_bqml_getting_started(random_model_id):
     # [END bigquery_dataframes_bqml_getting_started_tutorial_evaluate]
 
     # [START bigquery_dataframes_bqml_getting_started_tutorial_predict]
```

Comment:
There are two "predict" samples in https://cloud.google.com/bigquery/docs/create-machine-learning-model#use_your_model_to_predict_outcomes and https://cloud.google.com/bigquery/docs/create-machine-learning-model#predict_purchases_per_user

We'll need one for each. Let's disambiguate. I believe you are doing the first one, so `bigquery_dataframes_bqml_getting_started_tutorial_predict_by_country` may be a good more-specific region tag.